node/src/database/db_initializer.rs

```diff
@@ -20,6 +20,9 @@ use std::net::{Ipv4Addr, SocketAddr, SocketAddrV4};
 use std::path::Path;
 use tokio::net::TcpListener;
 
+#[cfg(test)]
+use std::any::Any;
```

Comment:
Is this still necessary? I don't see it being used in any of the tests in this file.